extras/syntax-highlighting/visual-code/nestml/package.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "nestml",
+    "displayName": "NESTML",
+    "description": "NEST simulator markup language for model code-generation",
```

Comment:
```suggestion
    "description": "NEST simulator modeling language for spiking neural network code generation",
```

Reply:
Is this how we describe NESTML? We don't really generate code for "spiking neural network" but for neuron and synapse models, though, don't we?

Reply:
Indeed, it's good to iterate a bit on this! This same phrase should re-appear in many places, e.g. the PyPI package, GitHub repository title, and so on.

"NESTML: A modeling language for spiking neurons and synapses"

Can we work the term "code generation" in there somewhere? Should we mention "point process"?

cc @jougs @abigailm

Reply:
@Silmathoron : we've come up with the following for now:

*NESTML: A concise modeling language and flexible code generator for spiking neuron and synapse models*

Would you be OK with this formulation?

Reply:
Sure, I'll try to include it tomorrow